Cargo.toml

```diff
@@ -23,23 +23,27 @@ default = []
 [dependencies.wgc]
 package = "wgpu-core"
 git = "https://github.com/gfx-rs/wgpu"
-rev = "7ce535cc97958d8e224247764f3d57c162594504"
+rev = "81b382f5cd7e371fc9df4a0b0fd438b759ce5027"
 # path = "../wgpu/wgpu-core"
-version = "0.8"
-features = ["raw-window-handle", "trace", "cross"]
+version = "0.9"
+features = ["raw-window-handle", "trace"]
 
 [dependencies.wgt]
 package = "wgpu-types"
 git = "https://github.com/gfx-rs/wgpu"
-rev = "7ce535cc97958d8e224247764f3d57c162594504"
+rev = "81b382f5cd7e371fc9df4a0b0fd438b759ce5027"
 # path = "../wgpu/wgpu-types"
-version = "0.8"
+version = "0.9"
 
 [dependencies]
 lazy_static = "1.1"
 raw-window-handle = "0.3"
 paste = "1.0"
 log = "0.4"
+[dependencies.naga]
+git = "https://github.com/gfx-rs/naga"
+rev = "0b9af95793e319817e74a30601cbcd4bad9bb3e6"
+features = ["wgsl-in", "spv-in"]
```

Comment:
shouldn't need this, since wgpu-native doesn't use `wgsl-in` directly. Only `spv-in` is needed here